controller/comment-controller.js

```diff
@@ -0,0 +1,48 @@
+const Comment = require("../models/comment");
+const Post = require("../models/post");
+
+const validation = require("../validators/validation");
+
+exports.addComment = async (req, res) => {
+  const { postId } = req.params;
+  const { description } = req.body;
+  const user = req.user._id;
```

Comment:
`userId`
Please be consistent on naming.

Reply:
Make this change everywhere.

Reply:
earlier it was userId, but you said to change it to user, hence changed..